io/pointer/pointer.go

```diff
@@ -36,6 +36,10 @@ type Event struct {
 	Position f32.Point
 	// Scroll is the scroll amount, if any.
 	Scroll f32.Point
+	// Scale is the rate of a pinch gesture
+	Magnification float32
+	// Rotation is the rate of a rotate gesture
+	Rotation float32
```

Comment:
I don't "rate" is right. According to the [documentation](https://developer.apple.com/documentation/appkit/nsevent/1526249-rotation) it seems rotation is a property of a tablet pen, and that it is in absolute degrees. If so, it seems more like a physical property (the angle of the pen) than a gesture (say, rotating a map with a two-finger gesture).